src/client/datascience/jupyter/kernels/types.ts

```diff
@@ -22,10 +22,9 @@ export type LiveKernelModel = IJupyterKernel &
     Partial<IJupyterKernelSpec> & { model: Session.IModel | undefined; notebook?: { path?: string } };
 
 export enum NotebookCellRunState {
-    Running = 1,
-    Idle = 2,
-    Success = 3,
-    Error = 4
+    Idle = 'Idle',
```

Comment:
Turned into strings so we can log these, else we just end up with numbers, 2,3,4